runtime/compiler/env/j9method.cpp

```diff
@@ -5974,6 +5974,12 @@ TR_ResolvedJ9Method::staticSignatureChars(I_32 cpIndex, int32_t & len)
    return cpIndex >= 0 ? fieldOrStaticSignatureChars(cpIndex, len) : 0;
    }
 
+J9UTF8 *
+TR_ResolvedJ9Method::fieldOrStaticSignature(I_32 cpIndex)
+   {
+   return J9ROMNAMEANDSIGNATURE_SIGNATURE(J9ROMFIELDREF_NAMEANDSIGNATURE(&romCPBase()[cpIndex]));
```

Comment:
We may need to add the test `cpIndex >= 0` like in `TR_ResolvedJ9Method::fieldOrStaticSignatureChars(I_32 cpIndex, int32_t & len)`